src/Date.php

```diff
@@ -103,23 +103,30 @@ public static function createFromFormat($format, $time, $timezone = null)
      * Alias for diffForHumans.
      *
      * @param  Date $since
-     * @param  bool $absolute Removes time difference modifiers ago, after, etc
+     * @param  bool $syntax  Removes time difference modifiers ago, after, etc
+     * @param  bool $short   (Carbon 2 only) displays short format of time units
+     * @param  int  $parts   (Carbon 2 only) maximum number of parts to display (default value: 1: single unit)
+     * @param  int  $options (Carbon 2 only) human diff options
      * @return string
      */
-    public function ago($since = null, $absolute = false)
+    public function ago($since = null, $syntax = null, $short = false, $parts = 1, $options = null)
     {
-        return $this->diffForHumans($since, $absolute);
+        return $this->diffForHumans($since, $syntax, $short, $parts, $options);
     }
 
     /**
      * Alias for diffForHumans.
      *
      * @param  Date $since
+     * @param  bool $syntax  Removes time difference modifiers ago, after, etc
+     * @param  bool $short   (Carbon 2 only) displays short format of time units
+     * @param  int  $parts   (Carbon 2 only) maximum number of parts to display (default value: 1: single unit)
+     * @param  int  $options (Carbon 2 only) human diff options
      * @return string
      */
-    public function until($since = null)
+    public function until($since = null, $syntax = null, $short = false, $parts = 1, $options = null)
```

Comment:
Same remark as `ago`.

Reply:
With `until` it's required because it already exist in Carbon 2 with this parameters.